data-prepper-plugins/common/src/main/java/org/opensearch/dataprepper/plugins/sink/LatencyMetrics.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.plugins.sink;
+
+import org.opensearch.dataprepper.metrics.PluginMetrics;
+import io.micrometer.core.instrument.DistributionSummary;
+import org.opensearch.dataprepper.model.event.EventHandle;
+
+import java.time.Duration;
+import java.time.Instant;
+
+public class LatencyMetrics {
```

Comment:
If we are able to move the tracking of this into `data-prepper-core`, please move this class into `data-prepper-core`. There should be no need to expose it in that situation.